tm2/pkg/std/memfile.go

```diff
@@ -2,99 +2,231 @@ package std
 
 import (
 	"fmt"
+	"io/ioutil"
+	"path/filepath"
 	"regexp"
+	"slices"
 	"sort"
 	"strings"
 )
 
+// XXX rename to mempackage.go
+
+const fileNameLimit = 256
```

Comment:
Can we consider a lower limit, especially since it's only the file part that can be added to an already long path or URL?